vcap_utils/vcap_utils/backends/base_openvino.py

```diff
@@ -61,11 +65,10 @@ def __init__(self, model_xml: os.PathLike, weights_bin: os.PathLike,
     @classmethod
     def from_bytes(cls, model_bytes: bytes, weights_bytes: bytes,
                    *args, **kwargs):
-        with NamedTemporaryFile() as model_fi, \
-                NamedTemporaryFile() as weights_fi:
-            model_fi.write(model_bytes)
-            weights_fi.write(weights_bytes)
-            return cls(model_fi.name, weights_fi.name, *args, **kwargs)
+        """This method is deprecated. Use the BaseOpenVINOBackend constructor
+        instead.
+        """
+        return cls(model=model_bytes, weights=weights_bytes, *args, **kwargs)
```

Comment:
Doesn't have to be this review, but should we have a "deprecated" decorator that will log when something is deprecated, a la Tensorflow?

Reply:
Good idea. I guess I'll use the Sphinx format since that seems to be what we've standardized on and it's probably what we'll use if we ever write auto-generated API documentation.